api/lib/tx-to-rest-converter.js

```diff
@@ -3,57 +3,49 @@
 var ripple = require('ripple-lib');
 var utils = require('./utils');
 var _ = require('lodash');
-var Promise = require('bluebird');
 var constants = require('./constants');
 var parseBalanceChanges = require('ripple-lib-transactionparser')
                           .parseBalanceChanges;
 var parseOrderBookChanges = require('ripple-lib-transactionparser')
                             .parseOrderBookChanges;
 
-function TxToRestConverter() {}
-
 // This is just to support the legacy naming of "counterparty", this
 // function should be removed when "issuer" is eliminated
-function renameCounterpartyToIssuer(orderChanges) {
+function renameCounterpartyToIssuerInOrderChanges(orderChanges) {
   return _.mapValues(orderChanges, function(changes) {
     return _.map(changes, function(change) {
+      return utils.renameCounterpartyToIssuerInOrder(change);
```

Comment:
Then almost all of my commits messages would end with "& some miscellaneous refactoring". I don't think it's necessary.